xml/article_setup.xml

```diff
@@ -1221,7 +1221,7 @@ rmdir /dev/cpuset/$1
     <term><literal>0x00000003</literal></term>
     <listitem>
      <para>
-      is processor #0 and processor #1.
+      is processors #0 and #1.
```

Comment:
This list looks weird. This is what the output of that would look like:
```
0x00000001.
    is processor #0.

0x00000002.
    is processor #1.

0x00000003.
    is processors #0 and #1.
```

I would suggest just using a normal itemizedlist for this, which allows for creating proper sentences:

```
<itemizedlist>
[...]
  <listitem>
    <para>
      <literal>0x00000003</literal> is processors #0 and #1.
    </para>
  </listitem>
</itemizedlist>
```

Reply:
I have replaced "is" with "means" as a simpler alternative. Acceptable?

Reply:
`means` indeed sounds better.

(My comment was about the XML structure though.)

Reply:
Yes,  I got that. I just try as a general rule to make the minimum viable changes. I think it's clear.

I did consider "denotes" but changed it again as I thought the word might be a bit obscure.

Reply:
I reconsidered and made it an [itemizedlist] as you suggested.